ghcup.cabal

```diff
@@ -65,44 +66,44 @@ flag tar
 
 common app-common-depends
   build-depends:
-    , aeson                  >=1.4
-    , aeson-pretty           ^>=0.8.8
-    , async                  ^>=2.2.3
-    , base                   >=4.12     && <5
-    , bytestring             >=0.10     && <0.12
-    , containers             ^>=0.6
-    , deepseq                ^>=1.4
-    , directory              ^>=1.3.6.0
-    , filepath               >=1.4.101.0
-    , variant                ^>=1.0
-    , megaparsec             >=8.0.0    && <9.3
-    , mtl                    ^>=2.2
+    , aeson                 >=1.4
```

Comment:
what's the point of reformatting here?

Reply:
I'd noticed the spacing had ended up inconsistent after the last couple of edits there, and just blasted it with `cabal-fmt`. I can either drop this reformatting, take the less-noisy option of just realigning the outliers, or reformat it with [gild](https://taylor.fausak.me/2024/02/17/gild/) to avoid such issues in the future. Your choice. (Also, please let me know if you'd prefer me just dropping/squashing the reformatting commit or if you'd like me to revert the reformatting on top of the current tip)

Reply:
Just to make things concrete, I pushed a branch to my fork with the `gild`-formatted version of the cabal files. Let me know if it's in any way useful to you
https://github.com/hseg/ghcup-hs/tree/reformat-cabal

Reply:
the reformatting makes it painful to fix merge conflicts